pandas/sparse/scipy_sparse.py

```diff
@@ -54,7 +54,7 @@ def get_indexers(levels):
         def _get_label_to_i_dict(labels, sort_labels=False):
             """ Return OrderedDict of unique labels to number.
             Optionally sort by label. """
-            labels = Index(map(tuple, labels)).unique().tolist()  # squish
+            labels = Index(list(map(tuple, labels))).unique().tolist()  # squish
             if sort_labels:
```

Comment:
use ``lmap``, which imports from ``pandas.compat`` (and handles the 2/3 differences)